README.md

```diff
@@ -41,25 +41,17 @@ render() {
 
 |Name            |Type        |Default     |Description
 |----------------|------------|------------|--------------------------------
-|className       | `string`  | | Sets classname for IconButton component.
+|classes       | `object`  | | Sets classes for components.
```

Comment:
Don't worry about this table, we have a script to automagically generate it from the jsdoc in the code.

Reply:
Ok!